src/notebook_utils/startup.py

```diff
@@ -25,3 +25,9 @@
     spark_to_pandas,
     display_namespace_viewer,
 )
+from spark.cluster import (
+    check_api_health,
+    create_cluster,
+    delete_cluster,
+    get_cluster_status,
```

Comment:
IIUC users will be calling these functions?

Reply:
not likely. Only power users should call `create_cluster`. But I think it would be helpful to provide users the ability to call all of them.